ts/src/binance.ts

```diff
@@ -3693,108 +3697,6 @@ export default class binance extends Exchange {
         const clientOrderId = this.safeStringN (params, [ 'newClientOrderId', 'clientOrderId', 'origClientOrderId' ]);
         let response = undefined;
```

Comment:
@pcriadoperez should we move this logic inside `createOrderRequest` ?
![image](https://github.com/ccxt/ccxt/assets/43336371/e2033477-017d-4eb8-9d46-e898eabcad76)